### PR TITLE
chore(virtctl): Codify ownership of virtctl packages

### PR DESCRIPTION
**Pull request by 0xFelix**
### What this PR does

Codify the ownership of some virtctl packages by adding OWNERS files. The ownership is mostly derived from labels found on virtctl's functional tests with the exception of virtctl expose.

Before this PR:

virtctl sub packages do not have individual OWNERS.

After this PR:

virtctl sub packages have individual OWNERS.

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
None
```

### CODE REVIEW

File: pkg/virtctl/expose/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - sig-network-reviewers
+approvers:
+  - sig-network-approvers
+labels:
+  - sig/network
```
Comment (EdDev) [inline pkg/virtctl/expose/OWNERS:6]: I'm not supper happy about this one (I think we should not support this at all), but it does make sense and reflects reality.

**Review by EdDev — APPROVED** [review]

**Review by xpivarc — COMMENTED** [review]
/approve

**Comment by 0xFelix** [comment]
Since everybody agreed:

/hold cancel

**Comment by brianmcarey** [comment]
/override pull-kubevirt-e2e-k8s-1.31-sig-performance

Incorrectly triggered by the status reconciler when the lane was bumped.

**Pull request merged**